src/ocl/dropoutocl.cpp

```diff
@@ -290,25 +290,25 @@ void DropoutDescriptor::DropoutForward(const Handle& handle,
         kernels.front()(pstates,
                         dropout,
                         amp_scale,
-                        int(in_len[1]),
-                        int(in_len[2]),
-                        int(in_len[3]),
-                        int(in_len[4]),
+                        static_cast<int>(in_len[1]),
```

Comment:
It's mostly a question, but why do you prefer cast instead of constructor call?
Is it to be more explicit?

I'm trying to adapt current coding conventions and is it mean that we should always use `static_cast<int>()` instead of `int()`?

Reply:
Meanwhile `int(val)` is not a c-style cast, it's a constructor call. C-style cast is `(int)val`.

Reply:
@CAHEK7 It _looks_ like a ctor, but please don't let C++ to fool you -- it is still a C-style cast, pls see https://en.cppreference.com/w/cpp/language/explicit_cast. static_casts are more flexibile, readable, smell stronger ;) and have much less ambiguous semantics.